File: python-venv/find_the_missing_letter/kata_function/find_the_missing_letter.py
def find_missing_letter(list):
    previous = list[0]

    for i in range(1, len(list), 1):
        if ord(list[i]) - ord(previous) != 1:
            return chr(ord(list[i]) - 1)
        previous = list[i]

#https://www.codewars.com/kata/5839edaa6754d6fec10000a2/train/python

File: find-the-missing-letter/find-the-missing-letter.js
const findMissingLetter = (array) => {
    let previous = array[0];

    for(let i = 1; i <= array.length; i++){
        if(array[i].charCodeAt() - previous.charCodeAt() !== 1){
            const code = array[i].charCodeAt() - 1;
            return String.fromCharCode(code);
        };
        previous = array[i];
    };
};

module.exports = { findMissingLetter };

//https://www.codewars.com/kata/5839edaa6754d6fec10000a2/train/javascript
